src/app/components/new-activity-modal/new-activity-modal.component.html

```diff
@@ -0,0 +1,44 @@
+<div class="container-new">
+  <div class="close">
+    <button mat-icon-button mat-dialog-close class="close-button">
+      <mat-icon>
+        close
+      </mat-icon>
+    </button>
+  </div>
+  <div class="header-title">
+    <h2>Add new activity</h2>
+  </div>
+
+  <div class="container-body-new">
+    <mat-form-field>
+      <mat-label>Name</mat-label>
+      <input matInput type="text">
+    </mat-form-field>
+
+    <mat-form-field>
+      <mat-label>Frequency</mat-label>
+      <mat-select [formControl]="days" multiple>
+        <mat-select-trigger>
+          {{days.value?.[0] || ''}}
+          <span *ngIf="(days.value?.length || 0) > 1" class="additional-selection">
+            (+{{(days.value?.length || 0) - 1}} {{days.value?.length === 2 ? 'other' : 'others'}})
+          </span>
+        </mat-select-trigger>
+        <mat-option *ngFor="let day of dayList" [value]="day">{{day}}</mat-option>
+      </mat-select>
+    </mat-form-field>
+
+    <mat-form-field>
+      <mat-label>Time</mat-label>
+      <input type="time" matInput>
+    </mat-form-field>
+  </div>
+
+  <div class="container-save">
+    <button class="save" mat-fab extended color="primary" (click)="createNew()">
```

Comment:
Create new what? use meaningful names.

```suggestion
    <button class="save" mat-fab extended color="primary" (click)="createActivity()">
```